### PR TITLE
Move check_slivers to drcs for use elsewhere.

### PR DESCRIPTION
**Pull request by robbintt**
This can be used downstream for sliver management in the future.

### CODE REVIEW

File: common-summoning.lic [hunk 1]
```diff
@@ -90,6 +90,46 @@ module DRCS
     return nil
   end
 
+  def check_slivers(game_state)
```
Comment (wstampley) [inline common-summoning.lic:93]: Should this be in DRCS or DRCMM?

Comment (Dartellum) [reply common-summoning.lic:93]: It is summoning but we do have a `DRCMM`, `common-moonmage.lic` file. I believe it should go into there. Also, I assume you tested this and made sure when it calls this the `game_state` is known.

Comment (robbintt) [reply common-summoning.lic:93]: I colocated it with moonblade, if I move it to drcmm, I will probably move more. Changing this to draft for now.

**Comment by rpherbig** [comment]
A few thoughts:
1. This will have to be multiple PRs. The first adds function(s) to `common-*` files, the follow-up PRs then use those functions. The reason is that we have to give changes to common files a few to be picked up by players or it can cause problems.
2. If the code is MM-specific, I recommend putting it in common-moonmage. There is some logical overlap between MM and S, and arguably most of S could be "WM".

**Comment by KatoakDR** [comment]
@robbintt Can you share what kind of slivers management you're thinking of? 

I may be a bit short sighted on the utility of this function. To me it feels very combat-trainer specific to quickly create some slivers for use as TKT ammo.

As-is, I'm inclined to say the method is fine where it is right now until we have definitive plans to reuse it outside of combat-trainer.

Thanks!

**Comment by KatoakDR** [comment]
Closing due to inactivity. Feel free to reopen this pull request and address code review feedback if you'd like to pursue this further. Thanks!

**Pull request closed without merging**